x-pack/plugin/security/build.gradle

```diff
@@ -177,6 +177,12 @@ tasks.named('forbiddenApisMain').configure {
   signaturesFiles += files('forbidden/ldap-signatures.txt', 'forbidden/xml-signatures.txt', 'forbidden/oidc-signatures.txt')
 }
 
+tasks.named('forbiddenApisTest').configure {
```

Comment:
Test exemption to use com.sun.net.httpserver.* in tests. This is the same exemption applied in other modules.